### PR TITLE
README improvements

### PR DESCRIPTION
**Pull request by thecadams**

### CODE REVIEW

**Comment by thecadams** [comment]
@mathieud @antoineeudes PTAL - was hesitant to just push this to master, since you might not see it

**Comment by antoineeudes** [comment]
> @mathieud @antoineeudes PTAL - was hesitant to just push this to master, since you might not see it

That's great thanks!

File: README.md [hunk 1]
```diff
@@ -14,6 +14,23 @@ npm start
 npm run build
 ```
 
+## Deploying
+
+tl;dr:
+```
+npm i -g firebase-tools # only need this once
```
Comment (mathieud) [inline README.md:21]: Any reason to not have this as a dev dependency?

Comment (thecadams) [reply README.md:21]: `-g` installs it globally so it’s in the path.

**Pull request merged**